### PR TITLE
Cleaned up project on baseline mod.

### PR DESCRIPTION
**Pull request by rdd13r**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-all.zip
```
Comment (CaptainLugaru) [inline gradle/wrapper/gradle-wrapper.properties:3]: Can we upgrade the version of Gradle here?

File: tech_magic.mcreator [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "dependencies": [],
+    "dependants": [],
+    "mcreatorDependencies": [],
+    "currentGenerator": "forge-1.16.5",
+    "credits": "Created with help of MCreator, produced in IntelliJ IDEA",
+    "modElementsPackage": "tales.hacker.munchkin.minecraft",
+    "lockBaseModFiles": false
+  },
+  "mcreatorVersion": 202100118117
+}
```
Comment (CaptainLugaru) [inline tech_magic.mcreator:33]: Add new line?

**Review by CaptainLugaru — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (aseix) [reply gradle/wrapper/gradle-wrapper.properties:3]: Yes, lets do this in the next story?

File: tech_magic.mcreator (hunk 2 above)
Comment (aseix) [reply tech_magic.mcreator:33]: Good idea. This is generated file for now.

**Comment by CaptainLugaru** [comment]
Cool!

**Pull request merged**